magicctapipe/scripts/lst1_magic/semi_automatic_scripts/database_production/nsb_to_h5.py

```diff
@@ -0,0 +1,89 @@
+"""
+Bash scripts to run LSTnsb.py on all the LST runs by using parallel jobs
+
+Usage: python nsb_level.py (-c config.yaml)
```

Comment:
the description is a copy paste error from nsb_level.py

Reply:
Fixed, Sorry!